.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
 # Every PR gets a review from an internal Expensify engineer
 * @Expensify/pullerbear
+
+# Assign the Design team to review changes to our styles
+src/styles/ @Expensify/Design @Expensify/pullerbear
```

Comment:
NAB but you could also consider including the `assets/` folder to check the images and animations that are added or changed

Reply:
Oh I like that idea, will update!